oak-run/src/main/js/oak-mongo.js

```diff
@@ -401,7 +401,7 @@ var oak = (function(global){
      * @method removeCollisions
      * @param {string} path the path of a document
      * @param {number} clusterId collision markers for this clusterId will be removed.
-     * @param {number} (optional) limit maximum number of collision markers to remove, default is 1000000.
+     * @param {number} [limit=1000000] maximum number of collision markers to remove.
      * @returns {object} the result of the MongoDB update.
      */
     api.removeCollisions = function(path, clusterId, limit) {
```

Comment:
Similar to the other reference to java code, should we add something like
"This corresponds to DocumentNodeStore.cleanRootCollisions (which is part of a startup and normal background update)"